src/app.js

```diff
@@ -14,7 +14,7 @@ angular.module('app').config(['$routeProvider', function routeProvider($routePro
     })
     .when('/register', {
       templateUrl: './views/register.html',
-      controller: 'RegistrationController',
+      controller: 'RegistrationController as vm',
```

Comment:
the convention here is to call the controller as something you would call an instance of the controller, eg. `registration`, and only use `vm` inside the controller, instead of `this`